src/selectors/fridge.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2020
+ */
+
+import { createSelector } from 'reselect';
+import { UIDatabase } from '../database/index';
+import { chunk } from '../utilities/chunk';
+
```

Comment:
Maybe overkill, but could add a selector just for `selectFridge`, and then the other selectors can all `createSelector` with that as a dependency. 

1. Probably not worth it just for this, but would make some of the other selectors tidy one-liners, e.g.:

```
export const selectFridge = ({ fridge = {} }) => fridge;
export const selectFridgeID = createSelector([selectFridge], ({ id }) => id);
```

2. I think would also be an optimisation (other selectors will be memoised and only recalculated if fridge changes)?

Reply:
I need to revisit these selectors - but one main point of using an `id` selector is to be a 'base' selector for a bunch of other selectors to depend on, rather than a reference to an object which is less reliable, especially as it's a realm object!